searchlib/src/tests/features/constant/constant_test.cpp

```diff
@@ -36,28 +35,25 @@ Value::UP make_tensor(const TensorSpec &spec) {
 
 }
 
-struct ExecFixture
+class ConstantTest : public ::testing::Test
 {
+protected:
     BlueprintFactory factory;
     FtFeatureTest test;
-    ExecFixture(const std::string &feature)
-        : factory(),
-          test(factory, feature)
-    {
-        setup_search_features(factory);
-    }
+    ConstantTest();
+    ~ConstantTest() override;
     bool setup() { return test.setup(); }
     const Value &extractTensor(uint32_t docid) {
         Value::CREF value = test.resolveObjectFeature(docid);
-        ASSERT_TRUE(value.get().type().has_dimensions());
+        EXPECT_TRUE(value.get().type().has_dimensions());
```

Comment:
Is this required to hold to prevent the caller from crashing instead of failing the test gracefully? If so, maybe it should be a regular `assert` instead

Reply:
No. operator== makes transient tensor specs and comares those.